### DISCUSSION TITLE
support for bluetooth device paring

### DISCUSSION THREAD

**Discussion by myusrn** [LSPosed/MagiskOnWSALocal#270] [Q&A] [answered]
With magisk rooted wsa install i can run `adb connect 127.0.0.1:58526` to establish android debug connection to the virtual device/machine followed by `adb shell am start -n 'com.android.settings/.Settings'` to bring up the android settings UI where i can access connected devices / bluetooth pairing functionality. 

That said when i run pair device it doesn't seem to find any bluetooth earbuds or other bluetooth devices i might want to pair with wsa setup for purpose of using the bluetooth devices play store installable app for doing firmware updates and custom configuration settings on the device.

Is support for bluetooth device pairing even possible with magisk rooted wsa install?

**Comment by Howard20181** [comment, marked as answer]
Report to [Microsoft](https://github.com/microsoft/WSA/issues) .
